chapter3/index.md

```diff
@@ -25,11 +25,11 @@ But we don't want `PC`s and `NPC`s to live up to 32767 years, so let's give `age
 
 ```sdl
 type Vampire extending Person {
-  property age -> int16;
+  overloaded property age -> int16;
```

Comment:
I think the idea here was to remove `age` from `Person` and only add it to `Vampire`. But this needs to be better phrased for sure.

Reply:
Oh. I see it now. It makes sense. I was working through it linearly and got bitten by it. Alright. I'm just going to add a single line that explicitly states that the reader is to remove `age` from the abstract class. Thank you.